tests/Microsoft.Identity.Test.Unit/PublicApiTests/TelemetryClientTests.cs

```diff
@@ -98,15 +109,24 @@ public async Task AcquireTokenSuccessfulTelemetryTestAsync()
                 CreateApplication();
                 _harness.HttpManager.AddMockHandlerSuccessfulClientCredentialTokenResponseMessage();
 
-                // Acquire token interactively
+                // Acquire token interactively with scope
                 var result = await _cca.AcquireTokenForClient(TestConstants.s_scope)
                     .WithAuthority(TestConstants.AuthorityUtidTenant)
                     .ExecuteAsync(CancellationToken.None).ConfigureAwait(false);
 
                 Assert.IsNotNull(result);
 
                 MsalTelemetryEventDetails eventDetails = _telemetryClient.TestTelemetryEventDetails;
-                AssertLoggedTelemetry(result, eventDetails, TokenSource.IdentityProvider, CacheRefreshReason.NoCachedAccessToken);
+                AssertLoggedTelemetry(
```

Comment:
For the fuuture - it helps reviewers when the tests are easier to read. Keep prod code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), test code can be "more" WET to increase readability. 

For example, if the assertion was more explicit, smth like:

```
Assert.AreEqual(CacheLevel.L2.ToString(), telemetry["cache_level"]);
Assert.AreEqual("https:\\graph.microsoft.com", telemetry["resource"]);
```